pkg/parcacol/querier.go

```diff
@@ -543,6 +548,8 @@ func (q *Querier) QuerySingle(
 func (q *Querier) findSingle(ctx context.Context, query string, t time.Time) (arrow.Record, string, profile.Meta, error) {
 	requestedTime := timestamp.FromTime(t)
 
+	level.Info(q.logger).Log("bamboozling_requested_timestamp", requestedTime, "query", query)
```

Comment:
lol .. also I don't think we need this log line anymore

Reply:
I... thought I had deleted this one. We definitely don't need it. 🙈